addon/components/basic-dropdown-content.ts

```diff
@@ -50,11 +52,11 @@ export default class BasicDropdownContent extends Component<Args> {
 
 
   get destinationElement(): Element | null {
-    return document.getElementById(this.args.destination);
+    return document.getElementById(this.args.destination) ?? getOwner(this).rootElement.querySelector?.(`[id="${this.args.destination}"]`);
```

Comment:
The use of nullish coalescing operator `??` here means there is a new dependency on either `@babel/preset-env` or `@babel/plugin-proposal-nullish-coalescing-operator`.

If the consuming app doesn't already have that plugin, it will fail during build:

```
Build Error (UglifyWriter)

Unexpected token: operator (?)
```

Reply:
Actually I don't know what the root of the problem was... this turned into a little adventure for me. Had to upgrade from ember-cli-uglify 2.x to ember-cli-terser 4.x before it finally would build.

I'm sure it's something specific to my project's dependencies and not something this addon has to worry about. However, since this would be the first usage of `??` in this addon, maybe it's best to dodge the problem entirely and avoid using `??`.

Reply:
Glad you figured it our already!

This issue comes up quite a lot, but always boils down to either not having `config/target.js` not configured correctly, or using uglify, or both. `ember-cli-babel` already brings all what is need to correctly transpile it. 

UglifyJS is not maintained anymore AFAIK, and does not support latest ES features. So chokes on nullish coalescing and other recent language features when you haven't IE or other older browsers in your `target.js` causing Babel to not transpile those away. 

So switching to `ember-cli-terser` was the right thing to do, which is part of the app/addon boilerplate already for a long time. 

tl;dr: using latest ES features here should be perfectly fine IMHO!

Reply:
Ah, thank you for the explanation! I feel much better about going down that rabbit hole now.